pkg/cmd/dedicated/listclusters/list.go

```diff
@@ -0,0 +1,132 @@
+package listclusters
+
+import (
+	"context"
+	"fmt"
+	clustersmgmtv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/dump"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/connection/api/clustermgmt"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	kafkamgmtclient "github.com/redhat-developer/app-services-sdk-go/kafkamgmt/apiv1/client"
+	"github.com/spf13/cobra"
+)
+
+type clusterRow struct {
+	Name          string `json:"name" header:"Name"`
+	ID            string `json:"id" header:"ID"`
+	Status        string `json:"status" header:"Status"`
+	CloudProvider string `json:"cloud_provider" header:"Cloud Provider"`
+	Region        string `json:"region" header:"Region"`
+}
+
+type options struct {
+	outputFormat            string
```

Comment:
Might be good to allow configuring the output format

Reply:
In the meantime this is not used

Reply:
Yes, I started with the option to format the output in json/yml etc. 
obviously I should have cleaned this up once I changed the functionality